atc/metric/emit.go

```diff
@@ -156,11 +160,6 @@ func (m *Monitor) Initialize(logger lager.Logger, host string, attributes map[st
 	return nil
 }
 
-func (m *Monitor) Deinitialize(logger lager.Logger) {
-	close(m.emissions)
```

Comment:
Getting rid of this means that the `m.emitLoop()` goroutines will leak in the tests, right? I guess that's fine, though

Reply:
yup. that's a leak i'm willing to take on in order to pay down the test-inflicted damage of the extra method. we could actually move the concurrency part out of the metrics subsystem completely, and have the atccmd create the emitloop goroutine, which would be kinda cool. We'd then be responsible for calling `emitLoop` (or whatever public name it ends up getting) in tests, but I'd be OK with that since that would also make the tests fully synchronous.

I might take a stab at that another time.